### PR TITLE
Adds Krav Maga's Grab Break and Fixes Cross Counter

### PR DESCRIPTION
**Pull request by F-a-y-t-h**
PeevedPenguin's fixed form his drop box link at #18117

(Still learning how to github; anything broken?)

### CODE REVIEW

File: data/json/martialarts.json [hunk 1]
```diff
@@ -933,7 +934,7 @@
             {
                 "id" : "venom_snake_combo_initiate",
                 "name" : "Viper Ambush",
-                "description" : "You've lured 'em in!  Your next attack will be a Viper Bite.",
+                "description" : "You've lured 'em in! Your next attack will be a Viper Bite.",
```
Comment (illi-kun) [inline data/json/martialarts.json]: This is both incorrect and unrelated change and it shouldn't be done in current PR. We're using double whitespaces after end of each sentence since we're using monospace fonts.

File: data/json/martialarts.json [hunk 2]
```diff
@@ -946,7 +947,7 @@
             {
                 "id" : "venom_snake_combo_continue",
                 "name" : "Viper Lock",
-                "description" : "You bit true!  Your next attack will be the Viper Strike!",
+                "description" : "You bit true! Your next attack will be the Viper Strike!",
```
Comment (illi-kun) [inline data/json/martialarts.json]: The same thing here.

File: data/json/techniques.json [hunk 3]
```diff
@@ -129,8 +129,8 @@
         "crit_tec" : true,
         "weighting" : -250,
         "messages" : [
-            "Snicker-snack!  You behead %s",
-            "Snicker-snack!  <npcname> beheads %s"
+            "Snicker-snack! You behead %s",
```
Comment (illi-kun) [inline data/json/techniques.json]: an here.

Comment (Xnfinity) [inline data/json/techniques.json]: ugh didn't notice this when i compared to get rid of his tabs.. oops

**Comment by mugling** [comment]
6 commits with names such as `derp` are a candidate for squash merging

File: data/json/techniques.json [hunk 4]
```diff
@@ -267,7 +267,7 @@
         "name" : "Cross Counter",
         "min_unarmed" : 5,
         "unarmed_allowed" : true,
-        "crit_tec" : true,
+        "dodge_counter": true,
```
Comment (Coolthulhu) [inline data/json/techniques.json]: Pretty sure this isn't supposed to be a dodge counter. It requires an on-dodge buff.
